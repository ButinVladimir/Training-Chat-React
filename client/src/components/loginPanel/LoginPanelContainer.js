import { connect } from 'react-redux';
import LoginPanel from './LoginPanel';

const mapStateToProps = state => ({
  loginned: state.loginPanel.loginned,
});

const LogoutFormContainer = connect(mapStateToProps)(LoginPanel);

export default LogoutFormContainer;
