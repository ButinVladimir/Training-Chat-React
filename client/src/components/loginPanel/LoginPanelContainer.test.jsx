import React from 'react';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import { mount } from 'enzyme';
import LoginPanelContainer from './LoginPanelContainer';

describe('LoginPanelContainer', () => {
  let mockStore;

  beforeEach(() => {
    mockStore = configureStore([]);
  });

  it('renders login panel', () => {
    const store = mockStore({
      loginPanel: {
        login: 'login',
        loginned: true,
      },
    });
    const component = mount(
      <Provider store={store}>
        <LoginPanelContainer />
      </Provider>,
    );

    expect(component).toMatchSnapshot();
  });
});
