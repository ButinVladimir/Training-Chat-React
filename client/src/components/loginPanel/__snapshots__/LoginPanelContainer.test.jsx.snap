// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPanelContainer renders login panel 1`] = `
<div
  className="chat-login-panel-container"
>
  <undefined />
</div>
`;
